ZenodoBestPractices.md

```diff
@@ -34,9 +37,15 @@ Assign metadata using the guidelines below.
 1. **Login**
 * Use your GitHub or ORCID account or create a new login. Using GitHub will automatically associate it with your repository.
 2. **Upload**
+
 * Click on "Upload" on the browser header (between search bar and *Communities*).
-* Click on "New Upload" - green button on upper right hand side of window.
-Note that your previous data packages are also shown. You can select these and edit their metadata.
+
+
+* **New Package.** Click on "New Upload" - green button on upper right hand side of window.
+* **Versioning a Package.** Select the package from your Uploads directory. Click on "New Version"
+
+At any time, you can select an uploaded package and edit its metadata. However, you CANNOT change the package's files without creating a new version.
```

Comment:
yes, this will assign it a new doi and it will only update the files that are changed per the FAQs